sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -436,6 +436,18 @@ def create_job_description(self, job):
     # TODO(silviuc): Remove the debug logging eventually.
     logging.info('JOB: %s', job)
 
+  def get_job_metrics(self, job_id):
```

Comment:
Do we want to use "@retry.with_exponential_backoff()" here in order to retry transient failures?